### ISSUE TITLE
TensorFlow 版本

### ISSUE THREAD

**Issue by guishilike** [david8862/keras-YOLOv3-model-set#139]
我可以问一下你使用的tensorflow版本吗，deep_sort的generate_detections.py使用的是1.x吧

**Comment by david8862** [comment]
@guishilike  deepsort demo中的deep association metric model部分使用的是原作者训练的pb模型，所以只支持TF 1.x版本。抱歉疏忽了，我会在README中补充相应的说明

**Comment by guishilike** [comment]
跟踪车辆到视频边缘（主要是下方）的时候，预测框会发生畸变，此时跟踪很不稳定，这是什么原因

**Comment by david8862** [comment]
@guishilike  我的理解是SORT/DeepSORT等跟踪方法是基于Kalman Filter对物体bbox的大小和运动轨迹进行估计，从而实现连续帧的跟踪。当物体移动到图像边缘时，由于bbox的形状和运动受到画面边界的限制，无法再遵循之前的连续变化规律，从而可能导致Kalman Filter失效

P.S，我在generate_detections.py中增加了对tf版本的检查，现在已经可以兼容tf2.x版本
